pkg/query-service/pqlEngine/engine.go

```diff
@@ -42,22 +44,42 @@ func NewPqlEngine(config *pconfig.Config) (*PqlEngine, error) {
 
 	logLevel := plog.AllowedLevel{}
 	logLevel.Set("debug")
-	logger := plog.New(logLevel)
+
+	allowedFormat := promlog.AllowedFormat{}
+	allowedFormat.Set("logfmt")
+
+	promlogConfig := promlog.Config{
+		Level:  &logLevel,
+		Format: &allowedFormat,
+	}
+
+	logger := plog.New(&promlogConfig)
 
 	opts := pql.EngineOpts{
-		Logger:        log.With(logger, "component", "promql evaluator"),
-		Reg:           nil,
-		MaxConcurrent: 20,
-		MaxSamples:    50000000,
-		Timeout:       time.Duration(2 * time.Minute),
+		Logger:     log.With(logger, "component", "promql evaluator"),
+		Reg:        nil,
+		MaxSamples: 50000000,
+		Timeout:    time.Duration(2 * time.Minute),
+		ActiveQueryTracker: pql.NewActiveQueryTracker(
```

Comment:
just FYI for other reviewers ..   I have been noticing a new file queries.active when running query service. This probably creates it. 

https://github.com/prometheus/prometheus/blob/main/promql/engine.go#L225